src/utils/constants.ts

```diff
@@ -2,8 +2,8 @@ import { BigNumber } from 'ethers'
 
 export const UNIVERSAL_ROUTER_ADDRESS = (chainId: number): string => {
   switch (chainId) {
-    case 1: // mainnet
-      return '0x4C60051384bd2d3C01bfc845Cf5F4b44bcbE9de5'
+    case 1: // mainnet. UR v1.2, all others are v1
```

Comment:
Did we want to deploy again on polygon as well for seaport support? I forget

Reply:
yeah we will deploy on all chains actually - gonna be doing that later today! just shipped on mainnet first so we can test with the interface